src/client/src/lib.rs

```diff
@@ -102,15 +103,19 @@ pub use crate::admin::FluvioAdmin;
 pub use crate::client::Fluvio;
 
 /// The minimum VERSION of the Fluvio Platform that this client is compatible with.
-const MINIMUM_PLATFORM_VERSION: &str = "0.7.0-alpha.5";
+const MINIMUM_PLATFORM_VERSION: &str = "0.7.1-alpha.0";
 
-/// Creates a producer that sends events to the named topic
+/// Creates a producer that sends records to the named topic
```

Comment:
nit: "sends records or keys/values to the named topic" or something.

Reply:
When I think "record" I consider that to include both the key and the value. This seems like the shortest way to describe it, saying "records or key/values" seems clumsy, and this is the one-line doc description.

Reply:
:+1: